### ISSUE TITLE
Question: Why is the Microsoft.ProjectReunion.WinUI 0.5.0-prerelease package unlisted?

### ISSUE THREAD

**Issue by mrlacey** [microsoft/microsoft-ui-xaml#4484]
Why is the Microsoft.ProjectReunion.WinUI 0.5.0-prerelease package unlisted?

Nuget.org displays the following notice [on the package listing](https://www.nuget.org/packages/Microsoft.ProjectReunion.WinUI/).

>  The owner has unlisted this package. This could mean that the package is deprecated, has security vulnerabilities or shouldn't be used anymore.

This raises multiple questions, and causes FUD.
Rather than speculate:

Is the package unlisted because it is in preview? 
Or is there something else that we should know about?

Will it be listed once out off preview?

**Comment by Marv51** [comment]
My guess is that is is because is very difficult to create a working project by adding the nuget directly instead of using the templates provided by the VS extension. (Because of the quite advanced changes to project file, publishing profile, and packaging project.)

But yes, I would also be interested in the official reason behind this decision.

**Comment by StephenLPeters** [comment]
I think Marv51 has the right idea, but @MikeHillberg and @ryandemopoulos want to weigh in?

**Comment by Noemata** [comment]
Along the same lines, why the Preview designation for 0.5?  From what I remember, March was supposed to be the official release of v 0.5, its designation being a compromise of sorts since a true 1.x release was a ways off.  The "Preview" + sub version designations are rapidly becoming meaningless.  Why not skip back to old school subversions instead?

We've already had to face up to the fact that WinUI 3.x is actually UWP v 2.x minus a bevy of capabilities.  The "next" next marketing spin has lost its ability to captivate.  Perhaps it's time to accept the reality of WinUI 1.x and that we truly are at WinUI v 0.5.  That view of the v 0.5 designation is more than reasonable and aligns nicely with the facts.

**Comment by chingucoding** [comment]
You are mixing things up a bit here @Noemata. 

The latest release was part of *Project Reunion*, a combination of different libraries to make Windows development easier. Project Reunion had it's version 0.5 release. *WinUI 3.0* is part of Project Reunion (Project Reunion is a collection of different projects after all). So the most recent release is NOT a 0.5 release, it is the first WinUI 3.0 production ready release.

Also there is no UWP version 2.x, there is WinUI 2.x which is a *library* that is built on top of the UWP XAML framework. *WinUI 3.0* is the effort of moving UWP XAML out of the OS and shipping it as a separate library. Also, this allows to use the previously known UWP XAML framework outside of the sandbox. So saying WinUI 3.x is actually is UWP v 2.x is quite an oversimplification.

**Comment by Noemata** [comment]
@chingucoding , I'm quite familiar with the tech stack.  And the marketing spin that revolved around WinUI when it started.  And all the other XAML variants going all the way back to early betas of Avalon in 2005.

The scary thing for some of us is that WinUI is taking a lot (most) of dev resources away from UWP, and is also being positioned as the successor to at least the UI side of UWP.  Looking outside in, its clear WinUI 3.0 is supposed to be the next version of UWP's UI 2.x.

If there is any confusion, it may be among Microsoft staffers.

@chingucoding , your qualifier of "first WinUI 3.0 production ready release" is hard to square with both the nomenclature that has been synthesized to gain interest in WinUI and the state of the various components that are now associated with WinUI.

Just to be clear, WinUI 3 Preview 4 was a nice step forward.  Lots of lights could be turned on with that release.  It had an impressive level of compatibility with UWP.  There have been some steps backwards with the latest WinUI 0.5 release as it is now bundled with Project Reunion, despite some things being "stabilized".  I won't get into that here.

The bottom line, WinUI 3.0 and all it's prerelease designations are becoming hard to untangle.  Especially when declarations like "production ready release" get thrown into the mix.  Just my two cents worth, going from the present v0.5 designation up to 1.x will be how I'll gauge progress moving forward because it seems to correlate with the state of where things are now.  My opinion only.  You can choose to ignore it if you wish @chingucoding.

**Comment by chingucoding** [comment]
You might be surprised, but yes WinUI 3 as a UI framework is actually the successor of a UI framework, not a revamp of the entire UWP stack or the UWP model. I don't think the naming is great right now at all, WinUI 3 should be recognizable as such and not be bundled as "Project Reunion 0.5". 

Also it is "not my qualifier" but the official wording from the roadmap: https://github.com/microsoft/microsoft-ui-xaml/blob/b799f7188badc6d91a67ba8b7f7840239bdbfefc/docs/roadmap.md

> Oh, you might be surprised to find that @thomasclaudiushuber , the author of your suggested reading largely agrees with me.

Reading your comment, it sound's like you want WinUI 3,0 to be named 0,5 instead, reading Thomas' comment you will find out he understands why there is confusion but he does not say we should call it WinUI 0,5 like you do it. But well you do you, I won't call "WinUI 3.0" "WinUI 0.5", the only confusing thing here is for me the bundling inside Project Reunion. If you actually open the NuGet package on nuget.org and read the description it also says WinUI 3.0 making it even harder to call it WinUI 0.5.

**Comment by thomasclaudiushuber** [comment]
> Oh, you might be surprised to find that @thomasclaudiushuber , the author of your suggested reading largely agrees with me.

Hey @Noemata , I'm not sure on what I agree, as I haven't made any comment in this issue here. But I think you're referring to this issue here: https://github.com/microsoft/microsoft-ui-xaml/issues/4427. I've added a few more details there.🙂

I fully agree to what @chingucoding  wrote here. The main issue in my opinion is that WinUI 2.x and WinUI 3.0 are two different things, as @chingucoding described it here in this issue. By giving the package for WinUI 3.0 now a 0.5 version number, it's getting even harder for developers to understand that this is actually WinUI 3.0. With my comment in the other issue I want to make the team aware of this. Beside this version number, bundling it with project reunion also does not feel right to me.

**Comment by mrlacey** [comment]
My original intention in this issue is to question the _unlisted_ status.

Separate discussion about numbering at https://github.com/microsoft/microsoft-ui-xaml/issues/4508

**Comment by MikeHillberg** [comment]
The intent/goal is that you can just reference the ProjectReunion package. It doesn't have all the metadata right now though, that's in the dependent packages. Referencing just one of the metadata packages isn't supported, so in order to prevent them from showing up in search they're unlisted.

The plan is to reduce to a single complete ProjectReunion package and not have the other packages at all.
